lib/db.js

```diff
@@ -193,17 +199,42 @@ DB.prototype.get = function(domain, req) {
             return self._get(tableName, req, schema);
         });
     } else {
-        return self._get(tableName, req, self.schemaCache[tableName]);
+        return P.try(function() {
+            return self._get(tableName, req, self.schemaCache[tableName]);
+        });
+    }
+};
+
+DB.prototype._createGetQuery = function(tableName, req, schema, includePreparedForDelete) {
+    var extracted = dbu.extractGetParams(tableName, req, schema, includePreparedForDelete);
+    var key = JSON.stringify(req);
```

Comment:
Generally, JSON.stringify output is not deterministic. In v8, it serializes object properties in the order they were created, rather than alphabetically.

To avoid this, it might be worth using https://github.com/substack/json-stable-stringify, which we already use in the cassandra backend.